<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrainingWindow</class>
 <widget class="QMainWindow" name="GrainingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graining</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBrowse"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionParameters"/>
    <addaction name="actionOperations"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
  </widget>
  <action name="actionBrowse">
   <property name="text">
    <string>Browse</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="text">
    <string>Open parameters window</string>
   </property>
  </action>
  <action name="actionOperations">
   <property name="text">
    <string>Open operations window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
